packages/components/psammead-section-label/src/titles.jsx

```diff
@@ -59,9 +57,11 @@ const FlexRow = styled.span`
   }
 `;
 
-const FlexTextRow = styled(FlexRow).attrs({
+const FlexTextRow = styled(FlexRow)``;
+
+FlexTextRow.defaultProps = {
```

Comment:
What do you think - should this just be passed into the `FlexTextRow` component in the exported `LinkTitle`, rather than using defaultProps? 

I have used this strategy for the `aria-` values that were previously being passed in as attrs to `IndexLinkCta` and `SectionLabelLink`

Reply:
Yeah, I don't think this styled component `FlexTextRow` is actually needed, because there are no styles attached to it. `FlexRow` could be used with `role="text"` passed directly to it.

Reply:
I've pushed changes.

Reply:
Awesome thank you Simon!